sdk/authorization/arm-authorization/src/models/parameters.ts

```diff
@@ -38,7 +38,7 @@ export const apiVersion1: msRest.OperationQueryParameter = {
     required: true,
     isConstant: true,
     serializedName: "api-version",
-    defaultValue: '2018-01-01-preview',
+    defaultValue: '2015-07-01',
```

Comment:
we are going backwards for defaultValue?

Reply:
This seems weird. @sarangan12 - Can you please verify this with the content of the rest-api-spec?

Reply:
@ramya-rao-a @amarzavery No. We are NOT going backwards with this code change. If you look at the current src/models/parameters.ts file (which was generated 4 months ago), it has 4 versions defined:

1. 2015-06-01
2. 2018-01-01-preview
3. 2018-09-01-preview
4. 2018-07-01-preview

Before 3 months, with this [PR](https://github.com/Azure/azure-rest-api-specs/pull/6219/files#diff-ce1d771389f8a93c847cfa81cd0fe5deR165) one more version is added and the file in the current PR has 5 versions defined:

1. 2015-06-01
2. 2015-07-01
3. 2018-01-01-preview
4. 2018-09-01-preview
5. 2018-07-01-preview

Because of the way the code is generated and showing in the PR, it appears that we are going back on default value. But, we are not. It is just a list of version.... :-)